src/test/java/de/tum/in/www1/artemis/assessment/TutorEffortIntegrationTest.java

```diff
@@ -69,7 +69,7 @@ void initTestCase() {
         course = courseUtilService.createCourseWithTextExerciseAndTutor(TEST_PREFIX + "tutor1");
         exercise = course.getExercises().iterator().next();
         studentParticipation = studentParticipationRepository.findByExerciseId(exercise.getId()).stream().iterator().next();
```

Comment:
will there never be more than one participation for the given exercise?

Reply:
No: `createCourseWithTextExerciseAndTutor` only ever creates one.